### PR TITLE
Fix some `jquery` deprecation warnings

### PR DESCRIPTION
**Pull request by jeffwidman**
Ths is a follow-on PR to
https://github.com/pallets-eco/flask-debugtoolbar/pull/241 in order to fix the deprecation warnings:
* `bind()` replaced by `on()`
* `click(handler)` replaced by `on('click', handler)`

Three deprecation warnings remain, but they are caused by the `jquery.tablesorter.js` plugin, so need to be handled upstream:
* https://github.com/Mottie/tablesorter/issues/1787
* https://github.com/Mottie/tablesorter/pull/1786

Related:
* https://github.com/pallets-eco/flask-debugtoolbar/issues/166

### CODE REVIEW

**Review by greyli — APPROVED** [review]

**Comment by jeffwidman** [comment]
Thanks for the quick review @greyli 👍 

Since this is the final PR in a 🚂  of Jquery-related PR's, I'll hold off on merging this until these other PR's are reviewed/merged:
* https://github.com/pallets-eco/flask-debugtoolbar/pull/235
* https://github.com/pallets-eco/flask-debugtoolbar/pull/236
* https://github.com/pallets-eco/flask-debugtoolbar/pull/241

**Pull request merged**
